docs/pages/server-access/guides/host-user-creation.mdx

```diff
@@ -0,0 +1,126 @@
+---
+title: Configure Teleport to Create Host Users
+description: How to configure Teleport to automatically create transient host users.
+---
+
+Teleport's SSH Service can be configured to automatically create local Unix users
+upon login.
+
+Host users created by Teleport are transient and will be deleted at the end of an SSH
+session. This feature has overlap with the [PAM](./ssh-pam.mdx) integration, however
+it provides some additional convenience of not having to manually configure a PAM
+script to create and delete users.
+
+It also has benefits over manually creating users as it allows for separate host
+users for each member of an organization. This allows for more fine grained control
+of permissions on a given host.
+
+## Prerequisites
+
+(!docs/pages/includes/edition-prereqs-tabs.mdx!)
+
+- A running Teleport Node. See the [Server Access Getting Started Guide](../getting-started.mdx) for  how to add a Node to your Teleport cluster.
+- The following utilities should be available in the PATH for the Teleport SSH Service:
```

Comment:
```suggestion
- The following utilities should be available in the PATH for the Teleport SSH Service, since it must execute these commands in order to create transient users:
```
Just to make it more explicit why we require these utilities in Teleport's PATH